mondrian/src/main/java/mondrian/rolap/RolapResult.java

```diff
@@ -1298,6 +1298,7 @@ private void executeStripe(
                     point.setAxis(axisOrdinal, tupleIndex);
                     final int savepoint = revaluator.savepoint();
                     try {
+                        revaluator.setEvalAxes( true );
```

Comment:
Have you done any checks for performance impact?  This change occurs within executeStripe, and will result in additional evaluation time with each and every evaluated cell.
We unfortunately don't have a performance test suite, but it would be good to do at least some minimal testing to determine how big an impact this will have when querying large cellsets with intensive calcs.  (e.g. a large crossjoin intersecting a calc member which aggregates a set).